Feature/kas 3788 minister checkbox

sergiofenoll opened the pull request:
# :warning: merges into #1607 

Adds the checkbox layout for selecting ministers for filtering in search screens.

Related PRs:
- [ ] https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/361

sergiofenoll committed:
feat: add minister filter using checkboxes

sergiofenoll committed:
Merge branch 'search_rework' into feature/KAS-3788-minister-checkbox

ValenberghsSven committed:
Merge branch 'search_rework' of github.com:kanselarij-vlaanderen/frontend-kaleidos into feature/KAS-3788-minister-checkbox

ValenberghsSven commented:
failing tests here are mainly from the base branch, jenkins run ongoing with fixes

sergiofenoll committed:
fix: directly wrap checkboxes in Auk::CheckboxList

Otherwise the checkboxes with short labels might be displayed next to
each other, which we don't want.

ValenberghsSven reviewed (approved):
ignoring flaky tests, this one is good
